### DISCUSSION TITLE
yaml files & env vars

### DISCUSSION THREAD

**Discussion by pmpetit** [cybertec-postgresql/pgwatch2#710] [Q&A]
Hello,
in the doc, it is said : "Specify hosts you want to monitor and with which metrics / aggressivness in a YAML file or files, following the example config located at /etc/pgwatch2/config/instances.yaml or online [here](https://github.com/cybertec-postgresql/pgwatch2/blob/master/pgwatch2/config/instances.yaml). Note that you can also use env. variables inside the YAML templates!"

how do you declare env vars inside the yam template ?

thx

**Comment by pashagolub** [comment]
Hello,

according to [sources](https://github.com/cybertec-postgresql/pgwatch2/blob/9db408989ebd7e6bce9dc3464756e187cad386a5/pgwatch2/pgwatch2.go#L4368) values prefixed with `$` are replaced with ENV VAR values.
